zowe/zowe-cli
# Fixes for --help-examples

**What It Does**
This fix corrects several commands where the  **--help-examples** fails to include the group in the example.  This pull request will fix issues https://github.com/zowe/zowe-cli/issues/1715 and https://github.com/zowe/zowe-cli/issues/1865. 

**How to Test**
Issue the **--help-examples** command at the group level. It should now display the group properly. For example, **zowe zos-jobs --help-example**s should display the group and command in the examples. 


`  

    Submit the JCL in the USS file "/a/ibmuser/compile.jcl":

      $ zowe zos-jobs submit uss-file "/a/ibmuser/compile.jcl"

`

**Review Checklist**
I certify that I have:
